kafka-ui-react-app/src/components/common/Pagination/Pagination.styled.ts

```diff
@@ -1,55 +1,90 @@
 import styled from 'styled-components';
 import { Colors } from 'theme/theme';
+import { Link } from 'react-router-dom';
 
 export const Wrapper = styled.nav`
   display: flex;
   align-items: flex-end;
-  padding: 0 16px;
+  padding: 25px 16px;
   gap: 15px;
-  padding-top: 25px;
 
   & > ul {
     display: flex;
     align-items: flex-end;
-    & .pagination-link {
-      height: 32px;
-      &.is-current {
-        background-color: ${Colors.brand[50]};
-        border-color: ${Colors.brand[50]};
-      }
+
+    & > li:not(:last-child) {
+      margin-right: 12px;
     }
   }
+`;
+
+export const PaginationLink = styled(Link)<{ $isCurrent: boolean }>`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  height: 32px;
+  width: 33px;
+
+  border-radius: 4px;
+  border: 1px solid
+    ${({ $isCurrent, theme }) =>
+      $isCurrent
+        ? Colors.neutral[10]
```

Comment:
pls update our theme and add something like `theme.paginationStyles.borderColor.current`